doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1591,21 +1592,172 @@ Numeral notations
          function application, constructors, inductive type families,
          sorts, and primitive integers) will be considered for printing.
 
-      :n:`( warning after @numeral )`
+         .. note::
+            Numeral notations for parameterized inductive type can be
+            added by declaring an :ref:`abbreviation <Abbreviations>`
+            for the inductive which instantiates all parameters.
+
+            .. coqtop:: in reset
+
+               Definition of_uint (u : Numeral.uint) : list unit :=
+                 let fix f n :=
+                   match n with O => nil | S n => cons tt (f n) end in
+                 f (Nat.of_num_uint u).
+               Definition to_uint (l : list unit) : Numeral.uint :=
+                 let fix f n :=
+                   match n with nil => O | cons tt l => S (f l) end in
+                 Nat.to_num_uint (f l).
+
+               Notation listunit := (list unit) (only parsing).
+               Numeral Notation listunit of_uint to_uint : nat_scope.
+
+            .. coqtop:: all
+
+               Check 0.
+               Check 1.
+               Check 2.
+
+      :n:`via @qualid__ind using ( {+, @qualid @qualid } )`
```

Comment:
You won't be able to use `qualid__ind` here.  The tool doesn't support it.

Reply:
I made an alias to reference called `qualid__ind`, I hope there is nothing wrong with that.